salt/orch/migration.sls

```diff
@@ -0,0 +1,85 @@
+{#- Get a list of nodes seem to be down or unresponsive #}
+{#- This sends a "are you still there?" message to all #}
+{#- the nodes and wait for a response, so it takes some time. #}
+
+{%- set parallelism = salt['pillar.get']('parallelism', 3) %}
```

Comment:
I would use the same `default_batch` variable we have in `orch/kubernetes.sls`. Maybe it could be updated in both places and be obtained from the same pillar variable...

Reply:
updated